src/index.html

```diff
@@ -9,10 +9,57 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="/src/style.scss"
```

Comment:
Stylesheets should be linked with an `.css` extension, not `.scss`. SCSS is a preprocessor and needs to be compiled into CSS before being linked in an HTML file.